en/docs/develop/restapis/applications.yaml

```diff
@@ -1703,7 +1703,7 @@ paths:
         Retrieve all the metadata related to the custom auth protocol identified by the inboundProtocolId
       description: >
         This API provides the capability to retrieve all the metadata related to the custom auth protocol
-        identified by the inboundProtocolId. <br>
+        identified by the inboundProtocolId. The URL encoded inbound protocol name is used as inboundProtocolId.<br>
```

Comment:
Shall we change this sentence? 
The URL-encoded inbound protocol name is used as the inboundProtocolId.

Reply:
addressed in c550a1866e11f4d04499c74631cda6f636a934da